src/30_Advanced/SeatMap.html

```diff
@@ -0,0 +1,224 @@
+-->
+<!-- -->
+<!doctype html>
+<html ⚡>
+  <head>
+    <meta charset="utf-8">
+    <title>SeatMap</title>
+    <link rel="canonical" href="<%host%>/advanced/seatmap.html" >
+    <meta name="viewport" content="width=device-width,minimum-scale=1,initial-scale=1">
+    <script async src="https://cdn.ampproject.org/v0.js"></script>
+    <!-- ## Setup -->
+    <!-- Additionally used AMP components must be imported in the header. We use `amp-pan-zoom` to make the seatmap zoomable and scrollable -->
+    <script async custom-element="amp-pan-zoom" src="https://cdn.ampproject.org/v0/amp-pan-zoom-0.1.js"></script>
+    <!-- We use `amp-bind` for enabling seat selection -->
+    <script async custom-element="amp-bind" src="https://cdn.ampproject.org/v0/amp-bind-0.1.js"></script>
+    <!-- We use `amp-list` for showing a list of seats -->
+    <script async custom-element="amp-list" src="https://cdn.ampproject.org/v0/amp-list-0.1.js"></script>
+    <!-- We use `amp-mustache` for rendering the `amp-list` content -->
+    <script async custom-template="amp-mustache" src="https://cdn.ampproject.org/v0/amp-mustache-0.2.js"></script>
+    <style amp-custom>
+      :root {
+      --abe-light-pink: rgba(182, 8, 69, 0.5);
+      --abe-pink: rgb(182, 8, 69);
+      }
+      *:focus {
+      outline: none;
+      }
+      .seat {
+        stroke-width: 3;
+        stroke: rgb(0,0,0);
+        fill:white;
+      }
+      .unavailable {
+        fill: var(--abe-light-pink);
+        pointer-events: none;
+      }
+      .container {
+        margin: 0;
+        display: flex;
+        flex-direction: column;
+        background: white;
+        border-radius: 5px;
+      }
+      .seat.selected {
+        fill: var(--abe-pink);
+      }
+      .disabled-seat {
+        pointer-events: none;
+        cursor: default;
+      }
+      .content {
+        background: white;
+        flex-grow: 1;
+        display: flex;
+        flex-direction: column;
+        border-radius: 5px;
+      }
+      .disabled {
+        pointer-events: none;
+        opacity: 0.5;
+        cursor: default;
+      }
+      .hidden {
+        display: none;
+      }
+      .clickable {
+        cursor: pointer;
+      }
+      .zoom-button {
+        bottom: 60px;
+        right: 0px;
+        z-index: 1;
+        border-color: Transparent;
+        background-color: Transparent;
+        margin: 8px;
+        width: 40px;
+        height: 40px;
+        padding: 0;
+        border-width: 0;
+        box-shadow: 0 2px 4px 0 rgba(0, 0, 0, 0.25);
+        background-color: white;
+        border-radius: 3px;
+      }
+      .opaque {
+        opacity: 0.5;
+        filter: alpha(opacity=50);
+      }
+      #choose-seats-button {
+        padding-bottom: 15px;
+        padding-right: 10px;
+      }
+      .seat-map-svg {
+        box-sizing: border-box;
+        padding-top: 30px;
+      }
+      #pan-zoom {
```

Comment:
Is this needed?

Reply:
yes, at default pan-zoom center its content, that mnakes impossible to place the placeholder in the right place, so we need display:initial